content/blog/2019/01-09-state-of-inexor-2018.md

```diff
@@ -0,0 +1,52 @@
+---
+layout:     post
+title:      State of Inexor 2018
+date:       2019-01-09 17:00:00 +0200
+author:     Fohlen
+summary:    Current state of Inexor in 2018 and outlook to 2019.
+---
+
+Howdy folks! It has been mere two months since the last official news from our project, so I'll briefly cover what happened:
+
+- we've had another amazing hackathon in Stuttgart
+- decision has been made to restart the project
+- the website got a major overhaul and the wiki and all of our internal structure changed
+
+This is a long story short. If you wish to dig deeper into why, we shall begin with
+
+## Hackathon
+
+Misan offered his house for a very [nice hackathon](https://github.com/inexorgame/inexor-core/issues/608). It took place from 23rd to 25th of November in Stuttgart. We have made a lot of decisions during this time span which influenced the later parts that will be covered. Also it was a cozy, familliar event!
+
+![Hackathon photo](https://user-images.githubusercontent.com/3730888/51029227-07b39c80-1596-11e9-8951-6b8f3fc72b6a.jpg "Hackathon 2018")
+
+_From left to right: Hanni, Fohlen, westernheld, Hanack, Misan, a_teammate, Croydon_
```

Comment:
This is mixing player names and GitHub names. I would make this uniform and go with GitHub names

Reply:
I wasn't so sure how to handle this ^_^

Reply:
Damn. then I really need to change my github name once in a while :P I'd prefer to be named Misan in the context of Inexor.

Reply:
Fohlen even pinged the GitHub misan accidentally in one of the above commits 😁

Reply:
One a second though if different people prefer different names, maybe leave it that way 🤔 

Making it uniform could prevent confusion however as @Fohlen just showed with the ping 😄